ci/easymesh_cert_version

```diff
@@ -0,0 +1 @@
+5d6bdb9168c6c2dd0b4c10fe25303ce5e7f28cd7
```

Comment:
Is it possible to a comment line with a link to this sha in github?
And / or a comment that explains _why_ this sha was chosen.

Reply:
> Is it possible to a comment line with a link to this sha in github?
> And / or a comment that explains _why_ this sha was chosen.

This SHA1 will change all the time. It points to the easymesh_cert revision which prplMesh was tested with. Every time we will update the easymesh cert, we will update this SHA1. Comment is completely redundant here, not everything deserves a comment - in this case the file name is enough IMO.

Reply:
I wonder where I'll have a chance to follow at least some of the decisions that led to these rapid changes.
I thought that a comment in this specific file might help, but maybe with the commit message that changes it would be good enough too. 
The point is that I am looking for _easy access_: where this sha is taken from (https://???), why was it chosen (we needed feature XYZ to solve issue #ccc, etc.) 
And again, I care less about the specific position these comments are written, and more about the fact that they _will_ be written. 
E.g In this very commit or PR I don't see anything that explains the why, or that links to something that might answer the selection of this specific sha.

Reply:
@Ran you're right when we update the version we should ideally mention what has changed in a nutshell (why we need to update it).
I added both the link to the target repo and the reason why I chose this hash in the commit message.